ui/components/src/LineChart.tsx

```diff
@@ -290,11 +293,27 @@ export function LineChart({
   );
 }
 
-function getFormattedDate(value: number) {
-  const XAXIS_DATE_FORMAT = new Intl.DateTimeFormat(undefined, {
+function getStepInterval(data: number[]) {
```

Comment:
This only gets called when stepMs isn't passed from panel container (makes embedding the chart independently a little easier)